infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -61,6 +64,11 @@ public class ManagementAgent {
      */
     private static final Duration TRIGGER_INTERVAL = Duration.ofSeconds(10);
 
+    /**
+     * Compactor flag
+     */
+    private static final String COMPACTOR_FLAG = "--compactor-script";
```

Comment:
nit: can we call this `COMPACTOR_SCRIPT_PATH` if possible?

Reply:
Fixed